.github/data/sync-registry-config.yaml

```diff
@@ -0,0 +1,3 @@
+k8s.gcr.io/sig-storage:
+  images-by-semver:
+    pause: "= 3.7"
```

Comment:
probably need to rename this?

Reply:
Yeah, don't know how that ended up here.